dist/images/daemonset.sh

```diff
@@ -143,6 +144,9 @@ while [ "$1" != "" ]; do
   --egress-ip-enable)
     OVN_EGRESSIP_ENABLE=$VALUE
     ;;
+  --ovs-daemonset)
+    OVS_DAEMONSET=true
```

Comment:
@aojea should we keep the current default behavior of running OVS daemons in a container? This works just fine for the `local` gateway mode. So, anyone who is trying out OVN K8s CNI will get a good out of the box experience.

So, should this be named --no-ovs-daemonset and `kind.sh` can call --no-ovs-daemonset.

Reply:
yeah, absolutely

Reply:
@girishmg @aojea with --ovn-unprivileged-mode=no and --gateway-options="--gateway-interface=eth0", shared mode works out of the box as well AFAIK. If we moving in a direction of shared mode to be a default, I'd say we should default to these for kind too. One issue which we not fully solved is how to move IP back to the eth0 interface in case of  ovs container delete. We need a watcher that would be running in the host and would move IP back automatically if user desires... This is however is not an issue for kind/cicd case I suspect...